cross-modal-search/README.md

```diff
@@ -1,38 +1,26 @@
-[![Jina](https://github.com/jina-ai/jina/blob/master/.github/badges/license-badge.svg "Jina is licensed under Apache-2.0")](#license)
-[![Jina Docs](https://github.com/jina-ai/jina/blob/master/.github/badges/docs-badge.svg "Checkout our docs and learn Jina")](https://docs.jina.ai)
-[![We are hiring](https://github.com/jina-ai/jina/blob/master/.github/badges/jina-corp-badge-hiring.svg "We are hiring full-time position at Jina")](https://jobs.jina.ai)
-<a href="https://twitter.com/intent/tweet?text=%F0%9F%91%8DCheck+out+Jina%3A+the+New+Open-Source+Solution+for+Neural+Information+Retrieval+%F0%9F%94%8D%40JinaAI_&url=https%3A%2F%2Fgithub.com%2Fjina-ai%2Fjina&hashtags=JinaSearch&original_referer=http%3A%2F%2Fgithub.com%2F&tw_p=tweetbutton" target="_blank">
-  <img src="https://github.com/jina-ai/jina/blob/master/.github/badges/twitter-badge.svg"
-       alt="tweet button" title="👍Share Jina with your friends on Twitter"></img>
-</a>
-[![Python 3.7 3.8](https://github.com/jina-ai/jina/blob/master/.github/badges/python-badge.svg "Jina supports Python 3.7 and above")](#)
-[![Docker](https://github.com/jina-ai/jina/blob/master/.github/badges/docker-badge.svg "Jina is multi-arch ready, can run on differnt architectures")](https://hub.docker.com/r/jinaai/jina/tags)
+## Overview
+|  |  |
+| ------------- | ------------- |
+| Learnings | How to use image captions (short descriptions of the image content) to search for images. |
+| Used for indexing | Images + captions. |
+| Used for querying | Image caption, e.g. "A boy playing basketball". |
+| Dataset used | [Flickr 8k](https://www.kaggle.com/adityajn105/flickr8k) containing 8k image caption pairs. |
+| Model used | Both [CLIP: Contrastive Language-Image Pre-Training](https://arxiv.org/abs/2007.13135) and [VSE++: Improving Visual-Semantic Embeddings with Hard Negatives](https://arxiv.org/pdf/1707.05612.pdf) are demonstrated. |
 
-</p>
+In this example, Jina is used to implement a cross-modal search system.
+This example allows the user to search for images given a caption description.
+We encode images and its captions (any descriptive text of the image) in separate indexes.
+The `image index` is later queried using the `text embeddings`.
+It is also possible to do it the other way around and use the `image encoding` to search for similar `text-embeddings` (captions).
 
-In this example, `jina` is used to implement a cross-modal search system. This example allows the user to search for images given a caption description and to look for a caption description given an image. We encode images and its captions (any descriptive text of the image) in separate indexes, which are later queried in a `cross-modal` fashion. It queries the `text index` using `image embeddings` and query the `image index` using `text embeddings`. 
+
+<details>
+  <summary>Click to expand more details on cross-modal retrieval!</summary>
 
 **Motive behind Cross Modal Retrieval**
```

Comment:
Can you move this to the "Extra" section? We have this exactly for all the info that could be useful but not 100% necessary to run a Jina example

Reply:
7821babc1884fffcc2d8076878ade4c02b8f0bc7